### PR TITLE
Add Vagrantfile

### PR DESCRIPTION
**Pull request by ariarijp**
This file will save some hands-on attendees and us!

### CODE REVIEW

**Comment by kakakakakku** [comment]
Thanks 👍 

But it doesn't work, and my environments are here.

- Vagrant 1.9.7
- VirtualBox 5.0.30

At first `vagrant up`, this logs are occurred.

```
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
```

And more, I check `docker ps` into Vagrant but containers don't work.

```
ubuntu@ubuntu-xenial:~/redash-hands-on$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
```

Next `vagrant provision`, `git clone` is not idempotence.

```
==> default: fatal: destination path 'redash-hands-on' already exists and is not an empty directory.
```

Can you reproduce it ?

**Comment by ariarijp** [comment]
Hi, @kakakakakku,

Thank you for your review.
I will check it again.

Thanks,

**Comment by ariarijp** [comment]
Hi @kakakakakku,

Mysteriously, I couldn't reproduce that error.

Can I have more verbosely logs in your environment?

Also, can you check which version of `ubuntu/xenial64` Vagrant Box are you using?
I'm using `ubuntu/xenial64 (virtualbox, 20180105.0.0)`.
If you are using an old version, can you try to upgrade your Vagrant Box?

Thanks,

**Comment by ariarijp** [comment]
@kakakakakku How's it going?

Please feel free to ask me If you have any question about this PR.

Thanks,

**Comment by kakakakakku** [comment]
@ariarijp 
Thank you, it works good 👍 

Please fix commands with `sudo`.

```diff
-$ docker-compose -f docker-compose.yml run --rm server create_db
+$ sudo docker-compose -f docker-compose.yml run --rm server create_db
-$ docker-compose -f docker-compose.yml up
+$ sudo docker-compose -f docker-compose.yml up
```

This error is occurred.

```sh
vagrant@ubuntu-xenial:~/redash-hands-on$ docker-compose -f docker-compose.yml run --rm server create_db
ERROR: Couldn't connect to Docker daemon at http+docker://localunixsocket - is it running?

If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.

vagrant@ubuntu-xenial:~/redash-hands-on$ docker-compose -f docker-compose.yml up
ERROR: Couldn't connect to Docker daemon at http+docker://localunixsocket - is it running?

If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
```

**Comment by ariarijp** [comment]
Thank you for your review.

I couldn't reproduce that error, but I can make sense about it.
so, I fixed Vagrantfile, could you check that again?

Thanks,

File: Vagrantfile [hunk 1]
```diff
@@ -35,8 +35,8 @@ After provisioning, You should login with "vagrant ssh".
 Then, run below commands.
 
 $ cd ~/redash-hands-on
-$ sudo docker-compose -f docker-compose.yml run --rm server create_db
-$ sudo docker-compose -f docker-compose.yml up
+$ sudo docker-compose run --rm server create_db
```
Comment (ariarijp) [inline Vagrantfile]: These changes relate to #5 .

**Pull request merged**